site/docs/samples/notebook.md

```diff
@@ -229,15 +229,44 @@ while [[ $(kubectl get m4dapplication my-notebook -o 'jsonpath={.status.ready}')
 
 ## Read the dataset from the notebook
 
+In your **terminal**, run the following command to print the [endpoint](../../reference/crds/#m4dapplicationstatusreadendpointsmapkey) to use for reading the data. It fetches the code from the `M4DApplication` resource:
+```bash
+printf "$(kubectl get m4dapplication my-notebook -o jsonpath={.status.readEndpointsMap})"
```

Comment:
@cdoron any idea on how to concat the fields as part of the print?

Reply:
@roee88 , @revit13 : try adding the following after the `kubectl` command:
`| python -c "import json; import sys; a=list(json.loads(sys.stdin.read()).values())[0]; print(a['scheme']+'://'+a['hostname']+':'+str(a['port']))"`

Reply:
python is not guaranteed to be installed. Maybe extract individual fields to variables like
```bash
ENDPOINT_SCHEME=$(kubectl get m4dapplication my-notebook -o jsonpath={.status.readEndpointsMap.scheme})
```
And then printf from that

Reply:
@roee88 @cdoron I committed a new version based on your suggestion, please let me know if you think it can be improved, Thanks